### PR TITLE
fix: removed X & Y from toolbox.ts and replaced movBy to moveTo

### LINKED ISSUE #6280: Built-in flyouts position blocks with x and y values incorrectly
**Describe the bug**

If you define a toolbox where the block info has x and y properties, the blocks will be positioned incorrectly.

**To Reproduce**

1. Create a toolbox definition.
2. Add blocks to it
3. Give those blocks x and y properties (That are non-zero)
4. Observe how they are positioned incorrectly.

**Expected behavior**

The toolbox should ignore the x and y properties on blocks because it has its own gap/separator logic for positioning blocks.

**Additional context**

- The [BlockInfo type](https://github.com/google/blockly/blob/fa58cbaaf7456ab227c2214af5dd0e7025dc731c/core/utils/toolbox.ts#L19) should be updated to not include X and Y.
- Blockly has two kinds of flyouts built-in, horizontal and vertical. You should update the layout logic in both of them.
- The flyouts should be changed to use [moveTo](https://github.com/google/blockly/blob/fa58cbaaf7456ab227c2214af5dd0e7025dc731c/core/block_svg.ts#L414) [instead of moveBy](https://github.com/google/blockly/blob/61322294da70c770b32abb28b9d695af75f61a16/core/flyout_vertical.js#L250)

**Tips for Getting Started**
We have a lot of information about how to get started [here](https://developers.google.com/blockly/guides/contribute/get-started) including details about our commit message style and how to open a PR.

### PR DESCRIPTION
**Pull request by varshneydevansh**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] I ran `npm run format` and `npm run lint`

### Resolves

- Fixes #6280 

### Proposed Changes

The [BlockInfo type](https://github.com/google/blockly/blob/fa58cbaaf7456ab227c2214af5dd0e7025dc731c/core/utils/toolbox.ts#L19) should be updated to not include X and Y.
Blockly has two kinds of flyouts built-in, horizontal and vertical. You should update the layout logic in both of them.
The flyouts should be changed to use [moveTo](https://github.com/google/blockly/blob/fa58cbaaf7456ab227c2214af5dd0e7025dc731c/core/block_svg.ts#L414) [instead of moveBy](https://github.com/google/blockly/blob/61322294da70c770b32abb28b9d695af75f61a16/core/flyout_vertical.js#L250)

#### Behavior Before Change

The `BlockInfo type` was picking up the values of `x` and `y` henceforth the position of the Toolbox Element was getting changed.

#### Behavior After Change

After removing the `X` and `Y` from the `BlockInfo type` the Toolbox Element is now ignoring the value and introducing the chnage to the code in files 
- [core/flyout_horizontal.ts](https://github.com/google/blockly/pull/7333/files#diff-67129394789d0d6086c6c255000bb6db6c9a4ecfa66a3e3f8618a64b0f804d40)
- [core/flyout_vertical.ts](https://github.com/google/blockly/pull/7333/files#diff-b866da6441c5a41965db65339b45aaffa9036332a928bde9ba9e5f59823b14a3)

to use `moveTo` instead of `moveBy`

Ignoring the x and y properties on blocks would allow them to position correctly.

### Reason for Changes

Because, the blocks were using the `relative coordinates` instead of the `absolute coordinates`

### Test Coverage

Browser which I used is MS Edge Version 115.0.1901.188 (Official build) (64-bit)

### Additional Information

It's my first PR on this project, so I might make mistakes.

### CODE REVIEW

**Comment by varshneydevansh** [comment]
@BeksOmega can you help me with the changes which I made are correct or not?

**Comment by BeksOmega** [comment]
> @BeksOmega can you help me with the changes which I made are correct or not?

Hello :D Currently the review for this is assigned to @maribethb ! I'll ask her if she wants me take it =)

File: core/flyout_horizontal.ts [hunk 1]
```diff
@@ -285,7 +285,8 @@ export class HorizontalFlyout extends Flyout {
         } else {
           moveX = cursorX - tab;
```
Comment (BeksOmega) [inline core/flyout_horizontal.ts:286]: I don't think we actually need this `moveX` variable, but I'm not gonna ask you to change it! Just leaving a note for my future self.

**Review by BeksOmega — APPROVED** [review]
This LGTM =) Could you update the PR description to describe how you tested this? Once we have info about that and CI passes I'll merge this!

File: core/flyout_horizontal.ts (hunk 1 above)
Comment (varshneydevansh) [reply core/flyout_horizontal.ts:286]: I did try to do the testing manually, but couldn't get completely was the correct way of doing it or not?

Comment (BeksOmega) [reply core/flyout_horizontal.ts:286]: You can follow the reproduction steps from the [original issue](https://github.com/google/blockly/issues/6280#issue-1315272059) to see if it reproduces the bug or not =)

In your case you can go into the [playground.html](https://github.com/google/blockly/blob/develop/tests/playground.html) and change `getToolboxElement` to the following:

```js
    function getToolboxElement() {
        return {
          "kind": "flyoutToolbox",
          "contents": [
            {
              "kind": "block",
              "type": "controls_if",
              "x": 10,
              "y": 10
            },
            {
              "kind": "block",
              "type": "controls_whileUntil",
              "x": 30,
              "y": 30
            }
          ]
        };
      }
```

The "x" and "y" properties were what was problems before. So these should be ignored and you should see blocks that are aligned correctly =)

If you could take a screen shot to confirm how it looks that would be perfect!

**Comment by varshneydevansh** [comment]
I will do this in the morning, tho. I did try doing the original steps earlier and found no changes. Furthermore, I will do it in the morning with a fresh perspective.

**Comment by varshneydevansh** [comment]
Hi @BeksOmega ,

I tested it now and the newly modified proposed changes are working as they were expected. 

I am attaching the SS for both the modified code and the unchanged code with different value in [playground.html](https://github.com/google/blockly/blob/develop/tests/playground.html) and change `getToolboxElement`


- **Modified code -**

image

image


- **Unchanged code -**

image

image


image

File: core/flyout_horizontal.ts (hunk 1 above)
Comment (varshneydevansh) [reply core/flyout_horizontal.ts:286]: I have updated the PR remark regarding the testing, and the required SS are attached below in the comment.

**Pull request merged**